frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -67,7 +67,7 @@ describe("scenarios > dashboard", () => {
       .type("{selectall}Orders per year")
       .blur();
 
-    cy.wait("@updateDashboard");
```

Comment:
If you are removing the wait, remove the `intercept` as well.

Btw, nothing is stopping you from waiting for this request to come back AND to assert on a string in the UI after that.